ledger-service/http-json/src/main/scala/com/digitalasset/http/HttpService.scala

```diff
@@ -195,16 +193,14 @@ object HttpService {
         websocketService,
       )
 
-      ignoreConParam = (res: Future[HttpResponse]) => (_: Http.IncomingConnection) => res
```

Comment:
I don’t understand the changes here. How are they related?

Reply:
The con param was only used for extracting the URL/IP.

Reply:
ah makes sense, thanks for the explanation.